app/src/ui/lib/git-config-user-form.tsx

```diff
@@ -47,7 +47,7 @@ export class GitConfigUserForm extends React.Component<
     super(props)
 
     this.state = {
-      emailIsOther: !this.accountEmails.includes(this.props.email),
+      emailIsOther: this.props.email ? !this.accountEmails.includes(this.props.email) : false
```

Comment:
I think this would be the right check:
```suggestion
      emailIsOther:
        this.accountEmails.length > 0 &&
        !this.accountEmails.includes(this.props.email),
```